reinforcement_learning/bandits_recsys_movielens_testbed/bandits_movielens_testbed.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p>This notebook introduces three features to bandit&nbsp;algorithms that *make* them applicable to a broader set of real-world problems</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/949/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Changed, thanks!</p>